### PR TITLE
DevsecOps.md localised to Hindi

### LINKED ISSUE #1183: [hi] `Devsecops` for `Hindi`
#### Why this term needs to be updated? (briefly) 
- Translation of Devsecops into Hindi localization.

### PR DESCRIPTION
**Pull request by octonawish-akcodes**
### Describe your changes
Translated Devsecops in hindi

### Related issue number or link (ex: `resolves #issue-number`)
Closes #1183 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

File: content/hi/devsecops.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## यह कैसे मदद करता है
+
+DevSecOps टीम साइलो को तोड़ने पर ध्यान केंद्रित करता है और सुरक्षित, स्वचालित वर्कफ़्लोज़ के निर्माण को बढ़ावा देता है।
+सुरक्षा अनुप्रयोगों का चयन करते समय, संगठनों को इसका लाभ उठाना चाहिए
+स्वचालित CI/CD कार्यप्रवाह और नीति प्रवर्तन जो डेवलपर को सशक्त बनाते हैं।
+लक्ष्य अवरोधक बनना नहीं बल्कि सुरक्षा नीतियों को लागू करना है
+उपयोगकर्ताओं को अपने प्रोजेक्ट को आगे बढ़ाने के तरीके के बारे में सटीक जानकारी देते हुए।
+जब ठीक से लागू किया जाता है, तो एक संगठन बेहतर टीम संचार प्राप्त करेगा और
+सुरक्षा दुर्घटनाओं और संबंधित लागतों को कम करें।
```
Comment (Garima-Negi) [inline content/hi/devsecops.md:31]: The alignment of sentences can be fixed. The original paragraph has sentences in continuation, please fix the extra new line and spaces.

File: content/hi/devsecops.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: DevSecOps 
+status: Completed
+category: concept
```
Comment (Garima-Negi) [inline content/hi/devsecops.md:4]: Please use अवधारणा in place of concept

File: content/hi/devsecops.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+tags: ["methodology", "", ""]
+---
+## यह क्या है
+
+DevSecOps शब्द विकास, संचालन और सुरक्षा जिम्मेदारियों के सांस्कृतिक विलय को संदर्भित करता है।
+यह सुरक्षा प्राथमिकताओं को शामिल करने के लिए [DevOps](/devops/) दृष्टिकोण का विस्तार करता है
+डेवलपर और परिचालन कार्यप्रवाह में न्यूनतम या बिना किसी व्यवधान के।
```
Comment (Garima-Negi) [inline content/hi/devsecops.md:11]:
```suggestion
यह सुरक्षा प्राथमिकताओं को शामिल करने के लिए डेवलपर और परिचालन कार्यप्रवाह में न्यूनतम या बिना किसी व्यवधान के, [DevOps](/devops/) दृष्टिकोण का विस्तार करता है।
```

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Comment by octonawish-akcodes** [comment]
check now @Garima-Negi

File: content/hi/devsecops.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+category: अवधारणा
+tags: ["methodology", "", ""]
+---
+## यह क्या है
+
+DevSecOps शब्द विकास, संचालन और सुरक्षा जिम्मेदारियों के सांस्कृतिक विलय को संदर्भित करता है।
+यह सुरक्षा प्राथमिकताओं को शामिल करने के लिए डेवलपर और परिचालन कार्यप्रवाह में न्यूनतम या बिना किसी व्यवधान के, [DevOps](/devops/) दृष्टिकोण का विस्तार करता है।
+DevOps की तरह, DevSecOps एक सांस्कृतिक बदलाव है, जिसे अपनाई गई तकनीकों द्वारा अद्वितीय अपनाने के तरीकों के साथ आगे बढ़ाया गया है।
+
+## समस्या यह संबोधित करता है
```
Comment (Garima-Negi) [inline content/hi/devsecops.md:13]: only ## समस्या is to be used here

File: content/hi/devsecops.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+DevOps प्रथाओं में [निरंतर एकीकरण] (/ निरंतर-एकीकरण /) और [निरंतर परिनियोजन] (/ निरंतर-वितरण /) शामिल हैं
+और अनुप्रयोग विकास और रिलीज चक्रों में तेजी लाएं।
+दुर्भाग्य से, स्वचालित रिलीज़ प्रक्रियाएं जो प्रतिनिधित्व करने में विफल होती हैं
+सभी संगठनात्मक हितधारक पर्याप्त रूप से मौजूदा मुद्दों को बढ़ा सकते हैं।
+एक प्रक्रिया जो सुरक्षा आवश्यकताओं पर विचार किए बिना नए सॉफ़्टवेयर को तेज़ी से रिलीज़ करती है
+किसी संगठन की सुरक्षा मुद्रा को नीचा दिखा सकता है।
+
+## यह कैसे मदद करता है
```
Comment (Garima-Negi) [inline content/hi/devsecops.md:22]: ## समाधान should be used here

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/hi/devsecops.md (hunk 5 above)
Comment (octonawish-akcodes) [reply content/hi/devsecops.md]: done

**Review by Garima-Negi — APPROVED** [review]

**Comment by seokho-son** [comment]
Hi @octonawish-akcodes 
Thank you for your contribution ! :) 
Please make sure to check the boxes in the checklist yourself. This is part of the template to obtain confirmation from the contributor.
image

(Hello Hindi team, @Garima-Negi @sayantani11 , could you please guide contributors to complete the checklist before a reviewer starts the review? Thank you!)

**Comment by octonawish-akcodes** [comment]
> Hi @octonawish-akcodes Thank you for your contribution ! :) Please make sure to check the boxes in the checklist yourself. This is part of the template to obtain confirmation from the contributor. image
> 
> (Hello Hindi team, @Garima-Negi @sayantani11 , could you please guide contributors to complete the checklist before a reviewer starts the review? Thank you!)

Done

**Review by anubha-v-ardhan — APPROVED** [review]

File: content/hi/devsecops.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+---
+## यह क्या है
+
+DevSecOps शब्द विकास, संचालन और सुरक्षा जिम्मेदारियों के सांस्कृतिक विलय को संदर्भित करता है।
+यह सुरक्षा प्राथमिकताओं को शामिल करने के लिए डेवलपर और परिचालन कार्यप्रवाह में न्यूनतम या बिना किसी व्यवधान के, [DevOps](/devops/) दृष्टिकोण का विस्तार करता है।
+DevOps की तरह, DevSecOps एक सांस्कृतिक बदलाव है, जिसे अपनाई गई तकनीकों द्वारा अद्वितीय अपनाने के तरीकों के साथ आगे बढ़ाया गया है।
+
+## समस्या 
+
+DevOps प्रथाओं में [निरंतर एकीकरण] (/ निरंतर-एकीकरण /) और [निरंतर परिनियोजन] (/ निरंतर-वितरण /) शामिल हैं
```
Comment (anubha-v-ardhan) [inline content/hi/devsecops.md:15]:
```suggestion
DevOps प्रथाओं में [निरंतर एकीकरण](/hi/continuous-integration//) और [निरंतर डिलीवरी](/hi/continuous-delivery/) शामिल हैं
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
Thank you @octonawish-akcodes

**Pull request merged**